oida/commands/linter.py

```diff
@@ -11,9 +11,13 @@ def print_violation(
 
 
 def run_linter(*paths: Path, checks: list[str]) -> bool:
+    print(f"paths: {paths}")
+    # for path in paths:
+    #    statistics = StatisticsGenerator(path=path)
     has_violations = False
     checkers = get_checkers(checks)
     for module in find_modules(*paths):
+        print(f"module > {module.path}")
```

Comment:
I'm assuming this is some leftovers that should be removed?

Reply:
Correct, I removed it just now.